decidim-participatory_processes/app/commands/decidim/participatory_processes/admin/destroy_participatory_process_type.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    module Admin
+      # A command with all the business logic when destroying a participatory
+      # process type in the system.
+      class DestroyParticipatoryProcessType < Rectify::Command
```

Comment:
```suggestion
      class DestroyParticipatoryProcessType < Decidim::Command
```
We're moving from Rectify::Command to Decidim::Command. See  #8759